src/main/resources/db/migration/V1_68__payment_type.sql

```diff
@@ -0,0 +1,2 @@
+DELETE FROM payment;
+ALTER TABLE payment ADD COLUMN payment_type VARCHAR(255) NOT NULL;
```

Comment:
Migrate data from somewhere instead. Or use a nullable field. Or use a default value.

Reply:
Thanks
Defaulting previous payments to SINGLE, keeping NOT NULL